Mandater/ClientApp/enums/ActionTypeEnums.ts

```diff
@@ -0,0 +1,12 @@
+﻿export enum ElectionActionEnum {
```

Comment:
ActionTypeEnums could be called ActionTypes instead, as it is implicit from folder structure what it is -- also cleans up usage.